Fix postinst script to handle disabled server

walles opened the pull request:
This change makes the postinst script source the /etc/default/drserv file and
not try to start the server unless it has been properly enabled.

Before this change the postinst script used to exit with an unhelpful error
message (and an exit code) when the server was disabled.

Commit:
Fix postinst script to handle disabled server

This change makes the postinst script source the /etc/default/drserv file and
not try to start the server unless it has been properly enabled.

Before this change the postinst script used to exit with an unhelpful error
message (and an exit code) when the server was disabled.

walles commented:
Review please?

@Shaywei, @nresare

Shaywei commented:
Tested locally and it works. +1

Shaywei commented:
I would also like to see a fix that adds the same echo message if you do a "service drserv start" instead of "start: Job failed to start" (when the ENABLED != yes).

Commit:
Log message if user tries to start disabled service

Writing to stdout / stderr is what we'd really like, but doing that from within
the upstart config file doesn't seem to be possible.

This change does the next best thing and puts a log message in
/var/log/upstart/drserv.log.

walles commented:
Remedied @Shaywei's suggestion as good as possible (log message, I wasn't able to print anything).
